en/identity-server/next/docs/guides/your-is/manage-console-access.md

```diff
@@ -1 +1,22 @@
+{% set wso2_is_console_access_note = "
+## Accessing the WSO2 Identity Server Console
+
+To access the WSO2 Identity Server Console, you need to have the required permissions and the URL of the Console. This can be achieved by navigating to the console through a web browser using the following URL format:
+
+   ```
+   https://<hostname>:<port>/console
+   ```
+
+For example, if you are using the default port, the URL will be `https://localhost:9443/console`.
+
+Upon navigating to this URL, you will be presented by the login screen of the WSO2 Identity Server. Use your credentials to log in to the Console.
```

Comment:
```suggestion
Upon navigating to this URL, you will be presented by the login screen of the {{product_name}}. Use your credentials to log in to the Console.
```

Reply:
Placeholders are not resolvable in the places we set the values to variables. Since this is IS specific page having the product name will not be a problem.